src/test-jdk17/java/com/automation/xmldoclet/simpledata/RecordClass.java

```diff
@@ -0,0 +1,20 @@
+package com.automation.xmldoclet.simpledata;
+
+/**
+ * Java 16 record class.
+ *
+ * @param x point in the X axis
+ * @param y point in the Y axis
+ */
+public record RecordClass(int x, int y) {
```

Comment:
Failing for now.
Awaiting discussion on possible ways to integrate.

Reply:
let's leave the implementation backwards compatible, so let's keep the toString() workaround.